script/DeployRehearsalContracts.s.sol

```diff
@@ -30,9 +30,6 @@ contract DeployRehearsalContracts is Deployer {
         council_safe = GnosisSafe(payable(vm.envAddress("COUNCIL_SAFE")));
 
         require(owner_safe.isOwner(address(council_safe)));
-
-        console.log("Deploying from %s", deployScript);
-        console.log("Deployment context: %s", deploymentContext);
```

Comment:
What were these logs for originally and why were they removed?

Reply:
cc @zchn who committed these logs, to confirm these were for debugging and not used by signers?

Reply:
Thanks! I'd advocate for keeping it in then unless there was a reason @0xfuturistic chose to remove it

Reply:
exactly. upon updating `lib/optimism`, the vars are missing so the lines give an error

Reply:
what does it make sense to use in replacement for `deployScript`? thought about using `Config.sig()`, but it defaults to string `string("run")`, which is not very descriptive @zchn